src/VisualStudio/Core/Def/Implementation/Interop/IComWrapperFixed.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.Interop
+{
+    [ComImport]
+    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    [Guid("CBD71F2C-6BC5-4932-B851-B93EB3151386")]
+    internal interface IComWrapperFixed
```

Comment:
Can we give a comment for what is 'fixed' here?

Reply:
Now updated